modules/core/04-channel/keeper/upgrade.go

```diff
@@ -363,6 +363,27 @@ func (k Keeper) WriteUpgradeAckChannel(ctx sdk.Context, portID, channelID string
 	emitChannelUpgradeAckEvent(ctx, portID, channelID, channel, upgrade)
 }
 
+func (k Keeper) WriteUpgradeConfirmChannel(ctx sdk.Context, portID, channelID string, counterpartyUpgrade types.Upgrade) {
```

Comment:
Should we write some short and sweet godoc prose for this beautiful function? ✍️